src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -76,23 +76,34 @@ public MembershipResults membershipResults(String currentUser, String uid) {
         // The disjoint of basis plus include and exclude: (Basis + Include) - Exclude
         List<String> groupingMembershipPaths = disjoint(parentGroupingPaths(basisIncludeExcludePaths),
                 parentGroupingPaths(excludePaths));
+        // A list of all group paths, in which the uhIdentifier is listed (including curated groupings), so we can find the intersection with curated groupings
```

Comment:
Sorry, I missed the window while this PR was still open, but for the future, can you limit each comment to column 80. I will probably result in more comment lines, but that's okay. Also, put a period at the end if you can.  No need to do this for the real code, though.